test/unit/object/test_base_object.py

```diff
@@ -21,6 +24,21 @@ def test_object_and_response(
     return test_objects_and_responses[request.param]
 
 
+@pytest.fixture(params=('same_file', 'same_folder', 'same_user', 'against_none', 'different_ids', 'different_types'))
+def objects_for_comparison(test_file, test_folder, mock_user, request, mock_box_session):
+    cases = {
+        'identical_object': (test_file, test_file, True),
+        'same_file': (test_file, File(mock_box_session, test_file.object_id), True),
+        'same_folder': (test_folder, Folder(mock_box_session, test_folder.object_id), True),
+        'same_user': (mock_user, User(mock_box_session, mock_user.object_id), True),
+        'against_none': (test_file, None, False),
+        'different_ids': (File(mock_box_session, '1'), File(mock_box_session, '2'), False),
+        'different_types': (File(mock_box_session, '1'), Folder(mock_box_session, '1'), False),
+        'not_api_object': (test_file, {'id': test_file.object_id, 'type': 'file'}, False)
```

Comment:
We prefer to use trailing commas (which is supported in Python in almost all cases), because then you can add new elements without affecting the diff of the previously-last line.